codeHF/workflows.yml

```diff
@@ -108,6 +119,11 @@ workflows:
       - o2-analysis-pid-tof-full
     tables: HFSELDPLUSCAND
 
+  o2-analysis-hf-lb-tolcpi-candidate-selector:
+    <<: *selector_3prong
```

Comment:
```suggestion
```
There is nothing that could be used from that inheritance

Reply:
Has been removed